### ISSUE TITLE
Elementary OS 8 multiple attempts at getting to start

### ISSUE THREAD

**Issue by gdulai** [elementary/os#740]
Hi!

Sorry if this is the wrong place to report this but this is what I think matches my issue the most.

I am facing the following problem: I installed ElementaryOs 8 many times now but it did not work so far. Most of my tries were met by black screen after trying to boot, which did not progress no matter how long I waited for. Now for the Nth attempt currently after install I am greeted by tty1 login screen but I was not able to create a user yet, so I cannot event attempt to fix something at this point. I was able to see some of the logs with pressing ESC and it seems that systemd.logind.service is unable to start.

Is there a default user I could try to fix things with or any other way to get past this?

Partitions:

Partitions 2gb fat32 /efi/boot
8gb sawp
150gb ext4 /
Specs:

Motherboard: AsRock B450M Steel Legend P10.30
CPU: AMD Ryzen 7 2700x
Memory: DDR4-2400 16GB x2
GPU: XFX AMD Radeon RX 6750 XT Core

**Comment by Mrrallepe** [comment]
same issue on an Lenovo V15
Systemd doesn't start
Sytem holds with a black screen

**Comment by Renzes** [comment]
Hi, 
Adding the "nomodeset" parameter in the "GRUB_CMDLINE_LINUX_DEFAULT" line to "/etc/default/grub" and running the "update-grub" command from LiveCD helped me.

**Comment by AceCodex** [comment]
> same issue on an Lenovo V15 Systemd doesn't start Sytem holds with a black screen

Me too. Lenovo ThinkBook 14+(2021).

**Comment by vjr** [comment]
See if this helps? -> https://github.com/orgs/elementary/discussions/633

**Comment by gdulai** [comment]
> See if this helps? -> https://github.com/orgs/elementary/discussions/633

So something similar helped. I was not able to edit the grub entry to remove the splash entry as the whole thing froze and was not reacting to anything anymore. 
I installed the ubuntu boot repair tool on the live image and with that I was able to remove the splash icon and crease the hide time to 10 seconds. 
I also tried mounting the partition and editing the /etc/default/grub but it would still required and update-grub which I was unable to run in that context.

I will close this issue as it was resolved.
